pkg/kubelet/userns/inuserns/inuserns_linux.go

```diff
@@ -16,9 +16,9 @@ limitations under the License.
 
 package inuserns
```

Comment:
This pkg can be probably removed

Reply:
I thought about it, but we don't know about external dependencies.

Deprecated it.

Reply:
Ah, I see this was only added recently (commit 2e999fff026) so makes sense to remove entirely.